File: Code coach/Medium/Average world length/average_word_length.py
import math

words = input().split(" ")
wordsNumber = 0
totalLength = 0

for word in words:
	wordsNumber += 1
	for char in word:
		if char.isalpha():
			totalLength += 1

averageLength = math.ceil(totalLength / wordsNumber)
print(averageLength)

File: Code coach/Medium/Average world length/average_word_length.rb
words = gets.chomp.split(" ")
wordsNumber = 0
totalLength = 0

words.each do |word|
	wordsNumber += 1
	word.each_char do |char|
		if char.match(/^[[:alpha:]]$/)
			totalLength += 1
		end
	end
end

averageLength = (totalLength.to_f / wordsNumber.to_f).ceil
puts averageLength
